korman/properties/modifiers/gui.py

```diff
@@ -680,6 +688,16 @@ def _create_moul_nodes(self, clickable_object, nodes, linkingnode, age_name, clk
             share.link_input(share_anim_stage, "stage", "stage_refs")
             share.link_output(linkingnode, "hosts", "shareBookSeek")
 
+        # Odds and Ends
+        stamped = nodes.new("PlasmaAttribBoolNode")
+        stamped.link_output(linkingnode, "pfm", "IsDRCStamped")
+        if not self.drc_stamp:
+            stamped.value = False
+        forcecam = nodes.new("PlasmaAttribBoolNode")
+        forcecam.link_output(linkingnode, "pfm", "ForceThirdPerson")
+        if self.third_person:
+            forcecam.value = True
+
```

Comment:
I'm thinking we should defer these changes to another PR.

Reply:
I agree.